src/webgl/p5.RendererGL.js

```diff
@@ -44,7 +44,8 @@ p5.RendererGL = function(elt, pInst, isMainCanvas) {
    * model view, projection, & normal
    * matrices
    */
-  this.uMVMatrix = new p5.Matrix();
+  this.uVMatrix = new p5.Matrix();
```

Comment:
Explained more in the general comment, but I'd prefer not to have separate view matrices and model matrices if possible.